Tasks/AppCenterDistributeV3/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -0,0 +1,48 @@
+{
+  "loc.friendlyName": "App Center Distribute",
+  "loc.helpMarkDown": "For help with this task, visit the Visual Studio App Center [support site](https://aka.ms/appcentersupport/).",
+  "loc.description": "Distribute app builds to testers and users via App Center",
+  "loc.instanceNameFormat": "Deploy $(app) to Visual Studio App Center",
+  "loc.releaseNotes": "Added support for multiple destinations.",
+  "loc.group.displayName.symbols": "Symbols",
+  "loc.input.label.serverEndpoint": "App Center service connection",
+  "loc.input.help.serverEndpoint": "Select the service connection for Visual Studio App Center. To create one, click the Manage link and create a new service connection.",
+  "loc.input.label.appSlug": "App slug",
+  "loc.input.help.appSlug": "The app slug is in the format of **{username}/{app_identifier}**.  To locate **{username}** and **{app_identifier}** for an app, click on its name from https://appcenter.ms/apps, and the resulting URL is in the format of [https://appcenter.ms/users/<b>{username}</b>/apps/<b>{app_identifier}</b>](https://appcenter.ms/users/{username}/apps/{app_identifier}). If you are using orgs, the app slug is of the format **{orgname}/{app_identifier}**.",
+  "loc.input.label.app": "Binary file path",
+  "loc.input.help.app": "Relative path from the repo root to the APK or IPA file you want to publish",
+  "loc.input.label.symbolsType": "Symbols type",
+  "loc.input.label.symbolsPath": "Symbols path",
+  "loc.input.help.symbolsPath": "Relative path from the repo root to the symbols folder.",
+  "loc.input.label.pdbPath": "Symbols path (*.pdb)",
+  "loc.input.help.pdbPath": "Relative path from the repo root to PDB symbols files. Path may contain wildcards.",
+  "loc.input.label.dsymPath": "dSYM path",
+  "loc.input.help.dsymPath": "Relative path from the repo root to dSYM folder. Path may contain wildcards.",
+  "loc.input.label.mappingTxtPath": "Mapping file",
+  "loc.input.help.mappingTxtPath": "Relative path from the repo root to Android's mapping.txt file.",
+  "loc.input.label.packParentFolder": "Include all items in parent folder",
+  "loc.input.help.packParentFolder": "Upload the selected symbols file or folder and all other items inside the same parent folder. This is required for React Native apps.",
+  "loc.input.label.releaseNotesSelection": "Create release notes",
+  "loc.input.label.releaseNotesInput": "Release notes",
+  "loc.input.help.releaseNotesInput": "Release notes for this version.",
+  "loc.input.label.releaseNotesFile": "Release notes file",
+  "loc.input.help.releaseNotesFile": "Select a UTF-8 encoded text file which contains the Release Notes for this version.",
+  "loc.input.label.isMandatory": "Require users to update to this release",
+  "loc.input.label.destinationType": "Release destination",
+  "loc.input.label.destinationGroupIds": "Destination IDs",
```

Comment:
Now that we select the destination type, can we make these more explicit? For example, this should be `Distribution Group IDs`. And `Destination ID` should be `Store ID`?

When we add Tester support at some point, we can add `Tester Emails`

Would be good to collect some PM feedbacks here.